tests/test_alembic.py

```diff
@@ -0,0 +1,11 @@
+from pytest_alembic.tests import (  # test_up_down_consistency, TODO: Review and fix the dowgrade migrations
+    test_model_definitions_match_ddl,
+    test_single_head_revision,
+    test_upgrade,
+)
+
+# trunk-ignore-all(pyright/reportUnusedExpression)
+# trunk-ignore-all(ruff/B018)
+test_model_definitions_match_ddl
```

Comment:
Hmm, rather than doing this, I think I'd prefer wrapping the imported tests inside a local test function.. Thoughts? @asacristani @diversemix 
```
def test_local_single_head_revision():
    tests.test_single_head_revision()

def test_upgrade():
    tests.test_upgrade()

def test_custom_model_definitions_match_ddl():
    tests.test_model_definitions_match_ddl()
```

Reply:
Or even calling the import functions directly using:

```
test_single_head_revision()
test_upgrade()
test_model_definitions_match_ddl()
```

Reply:
I would like them called explicitly too - then we actually know what is being done :+1: